common/containers/Home/index.jsx

```diff
@@ -23,16 +25,41 @@ export class Home extends Component {
     }
   }
 
+  handleNavigateUsers() {
+    if (__CLIENT__) {
+      window.location.href = '/users'
+    }
+  }
+
+  componentDidMount() {
+    this.constructor.fetchData(this.props.dispatch, this.props)
+  }
+
+  static fetchData(dispatch) {
+    dispatch(updateJWT())
+  }
+
   render() {
+    const isAdmin = this.props.isAdmin
     return (
-      <HomeComponent onEditProfile={this.handleEditProfile} onSignOut={this.handleSignOut}/>
+      <HomeComponent onEditProfile={this.handleEditProfile} onSignOut={this.handleSignOut} onNavigateUsers={this.handleNavigateUsers} isAdmin={isAdmin}/>
```

Comment:
Also, instead of using the role to determine whether or not the user should be able to see the view, use the `userCan` utility. That's what it's there for. :)